nodes/matrix/change_matrix_pivot.py

```diff
@@ -0,0 +1,77 @@
+import bpy
+from bpy.props import *
+from ... tree_info import keepNodeState
+from ... events import executionCodeChanged
+from ... algorithms.rotation import generateDirectionToRotationCode
+from ... base_types.node import AnimationNode
+
+pivotTypeItems = [
+    ("MATRIX", "Pivot Matrix", "Change Pivot by Matrix, like parent", "", 0),
+    ("VECTOR", "Pivot Location", "Change Pivot Location", "", 3),
+    ("LOC_ROT", "Center and Rotation", "Change Pivot by Center and Custom Rotation", "", 2),
+    ("AXES", "Center and Axes Direction", "Change Pivot by Center and Custom Rotation Axes", "", 1) ]
+
+class ChangeMatrixPivotNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_ChangeMatrixPivotNode"
+    bl_label = "Change Matrix Pivot"
+    
+    searchTags = [  ("Change Transform Matrix Pivot", {"pivotType" : repr("MATRIX")}),
```

Comment:
do we really need the search tags here? I think its easier to search for "change matrix pivot" only and then select the type from a dropdown list.